frontend/src/components/messages/MessagesTab.vue

```diff
@@ -1,136 +1,104 @@
-                  </v-btn>
-                  <v-btn class="btn-style" @click="false">
-                    <v-icon class="icon" left>mdi-email-open-outline</v-icon>
-                    <span class="btn-label">Mark read</span>
-                  </v-btn>
-                </div>
-              </v-col>
-            </v-row>
-          </template>
-          <!-- <template #headers="{ columns, isSorted, getSortIcon, toggleSort }">
-            <tr>
-              <th v-for="column  in  columns" :key="column.key" @click="toggleSort(column)"
-                :style="{ width: column.width }">
-                <div v-if="column.title === null">
-                  <v-checkbox v-model="checkBoxToggleAllState" @click="toggleAllCheckBoxHandler" hide-details
-                    density="compact"></v-checkbox>
-                </div>
-                <div v-else>
-                  {{ column.title }}
-                  <v-icon v-if="isSorted(column)">{{ getSortIcon(column) }}</v-icon>
-                </div>
-              </th>
-            </tr>
-          </template>
-          <template #item="{ item, index }">
-            <tr @click="rowClickHandler(item, index)"
-              :class="{ 'unread-notification': !item.selectable.isRead, 'highlighted': index === rowClickedIndex }">
-              <td :class="{ 'highlighted': index === rowClickedIndex }">
-                <v-checkbox @click="checkBoxClickHandler" v-model="checkBoxListState[index]" hide-details
-                  density="compact"></v-checkbox>
-              </td>
-              <td :class="{ 'highlighted': index === rowClickedIndex }">{{ (item.selectable.isRead) ? 'Read' : 'Unread' }}
-              </td>
-              <td :class="{ 'highlighted': index === rowClickedIndex }">{{ item.selectable.subject }}</td>
-              <td :class="{ 'highlighted': index === rowClickedIndex }">{{ item.selectable.dateReceived }}</td>
-            </tr>
-          </template> -->
-        </v-data-table-virtual>
+        <v-row>
+          <v-col class="mt-1">
+            <div class="flex-item">
+              <v-btn class="btn-style" @click="toggleNewRequestDialog()">
+                <v-icon class="icon" left>mdi-email-plus-outline</v-icon>
+                <span class="btn-label">New message</span>
+              </v-btn>
+              <v-btn class="btn-style" @click="toggleMarkUnreadButtonInMessageTable()">
+                <v-icon class="icon" left>mdi-email-outline</v-icon>
+                <span class="btn-label">Mark unread</span>
+              </v-btn>
+              <v-btn class="btn-style" @click="toggleMarkReadButton()">
+                <v-icon class="icon" left>mdi-email-open-outline</v-icon>
+                <span class="btn-label">Mark read</span>
+              </v-btn>
+            </div>
+          </v-col>
+        </v-row>
+        <v-row v-if="!assistanceRequests" class="mx-1">
+          <v-skeleton-loader type="table-tbody"></v-skeleton-loader>
```

Comment:
Not sure if it will work for you but I think it's normal to embed the content in the skeleton-loader itself. You could probably bind the loading property to assistanceRequests. See https://vuetifyjs.com/en/components/skeleton-loaders/#loading.